### DISCUSSION TITLE
Option to control mob tracking range (how far away mobs see you)

### DISCUSSION THREAD

**Discussion by Decme** [PurpurMC/Purpur#764] [__Dismissed__] [answered]
I was going through some fabric mods and came across this: https://www.curseforge.com/minecraft/mc-mods/farsighted-mobs

Would be cool to have this in Purpur, per-mob type (eg. the option to set skeletons & zombies tracking range to different values)

**Comment by YouHaveTrouble** [comment, marked as answer]
Already possible with [entities-target-with-follow-range](https://paper.readthedocs.io/en/latest/server/configuration.html#entities-target-with-follow-range) in paper.yml enabled and a datapack/plugin that modifies that vanilla attribute.

**Comment by Decme** [comment]
That would require writing a datapack or plugin, neither of which is easy for the average server owner.
